extensions/store/sql/identity-hub-store-sql/docs/schema.sql

```diff
@@ -0,0 +1,4 @@
+CREATE TABLE IF NOT EXISTS edc_identityhub
+(
+    item  VARCHAR NOT NULL UNIQUE
```

Comment:
I'm not quite sure about the schema to be honest. As I've alluded to in [this comment](https://github.com/eclipse-dataspaceconnector/IdentityHub/pull/43#discussion_r1000817845), even though IH objects are "schema-less", I would prefer to create some sort of `IdentityHubRecord` class that wraps the hub object, and provides things like an ID, date of entry, date of last update, etc.
For reference, EDC already provides the `MutableEntity` class, which could be extended here.
<br/>
Also, even though the `item` col is constrained to `NOT NULL UNIQUE`, it is not a primary key, which makes it difficult to search for and maintain the entries, for example Issue #45 will add some methods to the IH interface.
A PK is also `NOT NULL UNIQUE`, but in addition, it is automatically indexed.
<br/>
Lastly, once the `IdentityHubRecord` object is in place, I'd rename the data column to something like `payload` or `content`.